src/components/Header.js

```diff
@@ -65,7 +65,11 @@ const WorkWrapper = styled.button`
   padding: 0;
   background: ${({ theme }) => theme.colors.background};
   color: ${(props) =>
-    props.pathname === "/work"
+    props.pathname === "/work" ||
+    props.pathname === "/detailUXUI" ||
+    props.pathname === "/detailVideo" ||
+    props.pathname === "/detailFoto" ||
+    props.pathname === "/detailGraphic"
```

Comment:
You could do something like this:

```suggestion
    ["/work", "/detailUXUI", "/detailVideo", "/detailFoto", "/detailGraphic"].includes(props.pathname)
```

In addition, the array can be defined somewhere else.

Reply:
Thanks, I will go with that.